airflow/www/templates/airflow/plugins/dag_dependency_graph/dag_dependency_graph.html

```diff
@@ -0,0 +1,159 @@
+{% extends "airflow/master.html" %}
+
+{% block head_css %}
+{{ super() }}
+<link rel="stylesheet" type="text/css"
+    href="{{ url_for('static', filename='dagre.css') }}">
+<link rel="stylesheet" type="text/css"
+    href="{{ url_for('static', filename='graph.css') }}">
+
+<style>
+.graph {
+  background: #dbd8d6;
+  border: 2px solid black;
+  border-radius: 15px;
+  display: block;+  margin: auto;
+}
+
+.vpadding {
+  padding-bottom: 30px;
+  padding-top: 30px;
+}
+</style>
+{% endblock %}
+
+
+{% block body %}
+<div>
+  <h1>Dag Dependency Graph</h1>
+  <div class="vpadding">
+    <label>Search for dag:</label>
+    <select id="dag_dropdown" class="js-example-basic-single" name="state"></select>
+    <label>Direction:</label>
+    <select id="direction_dropdown" class="js-example-basic-single" name="state"></select>
+    <label>Degree: </label>
+    <select id="degree_dropdown" class="js-example-basic-single" name="state"></select>
+    <button type="button" onclick="refresh()">Search</button>
+  </div>
+  <svg class="graph" id="graph" width="100%" height="700">
+    <g id="dig"/>
+  </svg>
+  <div class="vpadding">
+    <a target="_blank" href="https://www.github.com/lyft/etl">Documention Link</a>
+  </div>
+</div>
+{% endblock %}
+
+
+{% block tail %}
+    {{ super() }}
+    <script src="{{ url_for('static', filename='d3.v3.min.js') }}"></script>
+    <script src="{{ url_for('static', filename='dagre-d3.js') }}"></script>
```

Comment:
@ArgentFalcon I put the code in incubator-airflow because I needed access to this dagre-d3 file. I tried importing it form etl but it didn't work. Do you think its possible from etl?

Reply:
I would think it should be possible. I'd have to play with it though. I'll look into it tomorrow.

Reply:
From slack, here's how to get it working in `ETL`:
Step 1:
Modify `etl/lib/lyftdata/airflow/plugins/blueprints/__init__.py` to look something like this:

```# Creating a flask blueprint to intergrate the templates and static folder
plugin_blueprint = Blueprint(
    "test_plugin", __name__,
    template_folder='templates',  # registers airflow/plugins/templates as a Jinja template folder)
    static_folder='static_plugin',
    static_url_path='/static/plugins'
)```

step 2:
Add a `static_plugin` folder under `blueprints`
step 3:
Put your Javascript file in the `static_plugin` folder
step 4:
Add the following line to your HTML:
```<script src="{{ url_for('static', filename='plugins/my_javascript.js') }}"></script>```
That should load correctly

Reply:
Sorry it took me so long.